### DISCUSSION TITLE
Will you make this responsive and mobile friendly in future ?

### DISCUSSION THREAD

**Discussion by d4eth** [JARVIS-AI/algocpu#5] [Q&A]
As right now the `TimeLog` function is not responsive.
Will you make it ?

**Comment by JARVIS-AI** [comment]
Exactly, Right now that part is not responsive, I will make it more platform friendly soon. 
I might need help.

**Comment by d4eth** [comment]
Bootstrap ?

**Reply by JARVIS-AI** [reply]
Yes, It is bootstrap 5
JavaScript and typescript

**Reply by d4eth** [reply]
Not going to be react or vuejs in future ?

**Reply by JARVIS-AI** [reply]
Interesting, I am making it in Vue.js too with my friend.

**Reply by d4eth** [reply]
Thanks for reply. If you need help I can do that
